deepchem/models/torch_models/dmpnn.py

```diff
@@ -7,38 +8,44 @@
 from deepchem.models.torch_models import layers
 from deepchem.models.torch_models import TorchModel
 
-try:
-  from torch_geometric.data import Data, Batch
-except ModuleNotFoundError:
-  raise ImportError("This model requires PyTorch Geometric to be installed.")
-
 from deepchem.feat import GraphData
 from deepchem.data import Dataset
 from typing import Union, List, Sequence, Optional, Iterable, Tuple
 
+logger = logging.getLogger(__name__)
 
-class _ModData(Data):
-  """
-  Modified version of Data class of pytorch-geometric to enable batching process to
-  custom increment values in certain keys.
-  """
+try:
+  from torch_geometric.data import Data, Batch
```

Comment:
I am not sure this a good practice to put an entire class under try-except block. A better way will be to check for torch_geometric in `torch_models/__init__.py` and if only torch geometric is present, import the module otherwise do not import the module.

Also, we can consider adding torch-geometric to CI.

Reply:
ok, I will update `torch_models/__init__.py`, thanks!